crates/bevy_reflect/src/list.rs

```diff
@@ -522,4 +523,24 @@ mod tests {
             assert_eq!(index, value);
         }
     }
+
+    // This test needs to be run in release mode
+    #[ignore]
```

Comment:
why not `#[cfg(not(debug_assertions))]`?

Reply:
I had never heard of that option, but it is a good idea! TIL :)